README.md

```diff
@@ -89,7 +108,12 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+An ERD (Entity Relationship Diagram) is a great tool to use when think about different relationship between data in a data base. Relationships can
+exist in many forms including: one-to-many, one-to-one, or many-to-one.
+
+The example above(Genie, Lamp; Person, Pet) is a perfect example of  relationships: A genie may come from one lamp; a lamp may belong to many
+people; a person may have on or many pets -- usually the pet only has one owner. The owner could only have one pet, which would make that a
+one-to-one relationship since pets usually only have one owner.
 ```
```

Comment:
That's a good definition of an ERD, but think a bit more about the relationships you outlined.

Person & Pet
* A person can have multiple pets. A pet (in this case) only has one owner. So what sort of relationship is that?